### ISSUE TITLE
fuse issues

### ISSUE THREAD

**Issue by koupatroupa** [kahing/goofys#232]
Hi

I'm trying to mount a bucket that causes fuse to fail when I'm try to run ls on it.
I've attached all relevant information in log.txt
The version of goofys I'm using is based off 3998020e9c5a38b01c414bf6b185a23012a80f52

[log.txt](https://github.com/kahing/goofys/files/1339186/log.txt)

**Comment by kahing** [comment]
could you try again and run goofys in the foreground (`-f`)? seems like the backtrace is lost

**Comment by koupatroupa** [comment]
I have to admit I wasn't honest with how I invoked goofys in the log.txt I attached.
I was doing
`./goofys --debug_s3 --debug_fuse bucket-root/subdir goofymnt`
I've attached the relevant parts of goofy's output when I run the above in foreground.

However, mounting without the subdir
`./goofys bucket-root goofymnt`
Works without issues.
The reason I was trying to mount the subdir is 'cuz I don't control that bucket and I thought I can't read the root. I guess that bucket's policy is more lenient than I thought.

Thanks for your time on this, I wouldn't have tried the `-f` if you hadn't suggested.

[goofyslog.txt](https://github.com/kahing/goofys/files/1356063/goofyslog.txt)
